includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -1323,6 +1323,8 @@ public function rest_endpoint_callback( $post_id, $route_to_call, $args = [] ) {
 
 		$image_scan_results = $this->scan_image( $image_url, [ $route_to_call ] );
 
+		print_r( $image_scan_results );
```

Comment:
I'm assuming this was here for testing purposes and should be removed

Reply:
Sorry for that, I don't usually forget debugging leftover code. Fixed.